src/libraries/Common/tests/TestUtilities/TestUtilities.csproj

```diff
@@ -68,7 +68,7 @@
     <PackageReference Include="xunit.core" Version="$(XUnitVersion)" ExcludeAssets="build" />
     <PackageReference Include="xunit.assert" Version="$(XUnitVersion)" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '$(NetCoreAppCurrentIdentifier)' and $([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) == '$(NetCoreAppCurrentVersion)'">
```

Comment:
```suggestion
  <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(NetCoreAppCurrent)', '$(TargetFramework)')">
```

Reply:
OK disregard. I'm unsure if this behavior works: https://github.com/dotnet/msbuild/issues/5171#issuecomment-717177639

Reply:
Why do we even need the intrinsic functions here? The `TargetFramework` isn't overloaded in this project with a suffix.

Reply:
> Why do we even need the intrinsic functions here? The TargetFramework isn't overloaded in this project with a suffix.

i removed it from version less projects.


```
<ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', '$(NetCoreAppCurrent)'))">
```

are u refering to use this everywhere ? bcoz in general it will be true for netcoreapp5.0 & less as well ?

Reply:
> are u refering to use this everywhere ? bcoz in general it will be true for netcoreapp5.0 & less as well ?

the code suggestion will only apply for `NetCoreAppCurrent`.